src/content/developers/docs/consensus-mechanisms/pos/weak-subjectivity/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Weak subjectivity
+description: An explanation of weak subjectivity and its role in PoS Ethereum.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+## Prerequisites
+
+To understand this page it is necessary to first understand the fundamentals of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/).
+
+## Weak Subjectivity
+
+Subjectivity in blockchains refers to reliance upon social information to agree on the current state. There may be multiple valid forks that are chosen from according to information gathered from other peers on the network. The converse is objectivity which refers to chains where there is only one possible valid chain that all nodes will necessarily agree upon by applying their coded rules. There is also a third state, known as weak subjectivity. This refers to a chain that can progress objectively after some initial seed of information is retrieved socially.
```

Comment:
Thoughts on switching these around and removing the weak subjectivity heading? We don't explicitly state it anywhere (we should), but this seems like the convention we lean towards most often: ([for example](https://ethereum.org/en/developers/docs/development-networks/))

```suggestion
Subjectivity in blockchains refers to reliance upon social information to agree on the current state. There may be multiple valid forks that are chosen from according to information gathered from other peers on the network. The converse is objectivity which refers to chains where there is only one possible valid chain that all nodes will necessarily agree upon by applying their coded rules. There is also a third state, known as weak subjectivity. This refers to a chain that can progress objectively after some initial seed of information is retrieved socially.

## Prerequisites {#prerequisites}

To understand this page it is necessary to first understand the fundamentals of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/).
```

Reply:
ok done in next commit